### ISSUE TITLE
IKU-02 and IKU-03 colliders 

### ISSUE THREAD

**Issue by TykoTek** [VenVen/Stock-Revamp#125]
The colliders on both of the inflatables extend well beyond the models when in deflated mode. If you strap an opposing pair of these onto the sides of a payload - say a small space station part - the fairing must be much larger in diameter than the actual part suggests because the colliders stick out so far.

Here are screen shots showing fairings being placed well past the ends of the inflatables and still not clearing the colliders.  https://imgur.com/a/G3Mxm

**Comment by TheToric** [comment]
do they extend to the uninflated size for the models?

**Comment by TykoTek** [comment]
No, the colliders extend maybe a meter past the end of the deflated model. Not even close to the full length of the inflated part.
